src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,125 @@
+---
+section: Guides
+chapter: Hardware
+title: Servo Setup and Calibration
+description: Learn how to test and calibrate servos.
+slug: /guides/hardware/servo-calibration
+---
+
+Servos are electromechanical components that allow for the precise control and positioning of the various limbs attached to the NUgus robot. Usually, when a servo is found to not be performing correctly (i.e. seizing or a crunching sound can be heard), it should be removed and replaced with a new servo.
+
+Before a servo is first used, its default parameters need to be updated to ensure correct operation. This is done using [Dynamixel Wizard](https://www.robotis.com/service/download.php?no=1670).
+
+## Set Up
+
+To start testing, calibrating or updating the parameters of a servo do the following:
+
+1. Connect the USB2Dynamixel to a USB port on the computer with Dynamixel Wizard.
+   ![Photo of the USB2Dynamixel attachment for Dynamixel Wizard connected to a computer](./images/servo_calibration/usb.jpg)
+
+2. Connect the power supply and the servo to the attached component of the USB2Dynamixel.
+   ![Photo of the servo cable connected to the power attachment of the USB2Dynamixel](./images/servo_calibration/power_supply_and_comms.jpg)
+
+3. Open Dynamixel Wizard.
+
+## Calibration
+
+To calibrate a servo perform the following instructions:
+
+1. Select "**Tools -> Calibration**". A window will pop up with instructions on how to calibrate the servo.
+
+2. Select the correct servo model. The model will be written on the servo (either MX-64 for head and arm servos or MX-106 for leg servos). Do **NOT** select the 2.0 versions.
+
+3. Select the correct communication port from the dropdown menu.
+
+4. Disconnect the servo from the USB attachment for a few seconds and then reconnect. If the software does not detect the disconnection, retry. If it fails, redo the calibration from step one. The software will then install the firmware to the servo.
```

Comment:
We have to install firmware to the servo to calibrate it? Is that correct? That seems strange to me, but I could just be totally out of the loop

Reply:
The calibration wizard will upload the firmware to the servo during the calibration process, doesn't require user intervention